src/app/modules/submission/email-submission/__tests__/email-submission.util.spec.ts

```diff
@@ -13,12 +13,7 @@ import { ObjectId } from 'bson'
 import { readFileSync } from 'fs'
 import { cloneDeep, merge } from 'lodash'
 
-import {
-  FieldResponse,
-  IAttachmentResponse,
-  SingleAnswerFieldResponse,
-  SPCPFieldTitle,
-} from 'src/types'
+import { SingleAnswerFieldResponse, SPCPFieldTitle } from 'src/types'
```

Comment:
relative import

Reply:
fixed in [06c5edc](https://github.com/opengovsg/FormSG/pull/6587/commits/06c5edcbdf73f320c4cbd2562172fbc81871bae5)